guides/v2.1/coding-standards/docblock-standard-javascript.md

```diff
@@ -212,7 +214,7 @@ project.SubClass.prototype.toString()&nbsp;{
 };
 </pre>
 
-<h4 id="frontend-dev-guide-javascript-js-coding-docblock-param">@param</h4>
+#### @param
```

Comment:
Please append `{#frontend-dev-guide-javascript-js-coding-docblock-param}` to preserve the custom id.